src/main/xar-resources/data/ant-tasks/ant-tasks.xml

```diff
@@ -728,11 +727,11 @@
         <para>The <code>addgroup</code> task adds a user group. </para>
         <programlisting language="xml" xlink:href="listings/listing-27.xml"/>
         <para>Attributes:</para>
-        <variablelist spacing="compact">
+        <variablelist>
             <varlistentry>
                 <term><code>name</code></term>
                 <listitem>
-                    <para>Name of the new group.</para>
+                    <para>Takes thename of the new group.</para>
```

Comment:
missing space between "the" and "name".

Also, either 'groupname' here and in the following <para>, or name in both cases. I see that you use 'username of the user' later on, so perhaps we want to keep groupname everywhere.